blueprint/api.apib

```diff
@@ -2534,10 +2594,34 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(array[Stripe Card Resource])
 + include JSON API Version
 
+## Stripe Customer Resource (object)
++ include Stripe Customer Resource Identifier
++ attributes(Stripe Customer Attributes)
++ relationships
+    + subscriptions
+      + data(Stripe Subscription Resource Identifier)
+    + user
+      + data(User Resource Identifier)
+
 ## Stripe Customer Resource Identifier (object)
 + id: `1` (string, required)
 + type: `stripe-customer`
 
+## Stripe Customer Attributes (object)
++ created: `016-07-08T03:03:51.967Z` (string) - A timestamp, indicating when the customer was created by Stripe
++ currency: `usd` (string)
++ delinquent: `true` (boolean) - Whether or not the latest charge for the customer’s latest invoice has failed.
++ email: `johndoe@email.com` (string)
++ `id-from-stripe`: `cus_9SBK9vXBkZEdfe` (string) - The ID that Stripe assigned the customer.
+
+## Stripe Customer Response (object)
++ data(Stripe Customer Resource)
++ include JSON API Version
+
+## Stripe Customers Response (object)
++ data(array[Stripe Customer Resource])
++ include JSON API Version
```

Comment:
Your alpha ordering is a little off here. I think Attributes => Resource => Resource Identifier => Response(s).